### PR TITLE
Add tiny-date-picker@2.0.3 with npm auto-update

### PR DESCRIPTION
**Pull request by armenavetisyan**
Pull request for issue: #10214

### CODE REVIEW

File: ajax/libs/tiny-date-picker/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "picker",
+    "chooser",
+    "calendar"
+  ],
+  "author": "Chris Davies <ferociousturtle@gmail.com>",
+  "license": "MIT",
+  "npmName": "tiny-date-picker",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```
Comment (pvnr0082t) [inline ajax/libs/tiny-date-picker/package.json]: There is no `dist` folder in the repo. Let's use `""` as basePath.

**Comment by PeterDaveHello** [comment]
@armenavetisyan any updates? Thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello Is that your `squash and rebase bot` mixed other commits?

**Comment by PeterDaveHello** [comment]
@pvnr0082t fixed, thanks.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PeterDaveHello** [comment]
@pvnr0082t can you send a new PR to close this one? Thanks.

**Comment by PeterDaveHello** [comment]
Ping @pvnr0082t

**Pull request closed without merging**